sparse_strips/vello_toy/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "vello_toy"
+version.workspace = true
```

Comment:
I think this is a package which we'll never release, and so it should not have a version.
See also that this package doesn't have `LICENSE-MIT`/`LICENSE-APACHE` files.

```suggestion
```